lib/ProductOpener/Producers.pm

```diff
@@ -421,6 +421,38 @@ fr => {
 
 );
 
+my %per_regexps = (
```

Comment:
I'm not quite sure why this is named `per_regexps`, as it doesn't container regular expressinos right now?

Reply:
Good point, I changed my mind and forgot to rename the field. :) I will rename it.